core/templates/dev/head/pages/exploration_player/ProgressDotsDirective.js

```diff
@@ -63,9 +63,9 @@ oppia.directive('progressDots', [
           });
 
           $scope.changeActiveDot = function(index) {
-            playerPositionService.setActiveCardIndex(index);
+            PlayerPositionService.setActiveCardIndex(index);
             $rootScope.$broadcast('updateActiveStateIfInEditor',
-                                  playerPositionService.getCurrentStateName());
+            PlayerPositionService.getCurrentStateName());
```

Comment:
Again as above please ident it once from `$rootScope`

Reply:
FAILED   Pattern checks failed. How to solve this

Reply:
@acashmkj , have you made the changes I requested? 
Also, I was wondering why haven't you made a separate branch for this issue ?